src/test/groovy/org/prebid/server/functional/tests/pricefloors/PriceFloorsFetchingSpec.groovy

```diff
@@ -1664,6 +1664,36 @@ class PriceFloorsFetchingSpec extends PriceFloorsBaseSpec {
         null               | JPY
     }
 
+    def "PBS should not receive floors log when the header has other controls"() {
```

Comment:
Test wording is strange. You're checking that everything works. And `max-age` also looks valid.